CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 History
 =======
 
+1.0.1 (2019-02-21)
```

Comment:
For the future: We should list changelog entrys under "master" and before we release we should change then "master" to the version and the date.  (And of course change the `VERSION` then too.)

Reply:
okay